src/cascadia/TerminalApp/Tab.cpp

```diff
@@ -83,10 +159,35 @@ void Tab::_Focus()
 // - delta: a number of lines to move the viewport relative to the current viewport.
 // Return Value:
 // - <none>
-void Tab::Scroll(int delta)
+void Tab::Scroll(const int delta)
 {
-    _control.GetControl().Dispatcher().RunAsync(CoreDispatcherPriority::Normal, [=](){
-        const auto currentOffset = _control.GetScrollOffset();
-        _control.ScrollViewport(currentOffset + delta);
+    GetLastFocusedTerminalControl().GetControl().Dispatcher().RunAsync(CoreDispatcherPriority::Normal, [=](){

```

Comment:
Let's capture `control` and not look it up again: there's a chance it could change between line 164 and 165 if the user shifts focus somehow